website/src/components/TaskPage/TaskPage.tsx

```diff
@@ -15,26 +16,30 @@ type TaskPageProps = {
 export const TaskPage = ({ type }: TaskPageProps) => {
   const { t } = useTranslation(["tasks", "common"]);
   const taskApiHook = taskApiHooks[type];
-  const { tasks, isLoading, reset, trigger, error } = taskApiHook(type);
+  const { response, isLoading, completeTask, skipTask } = taskApiHook(type);
   const taskInfo = TaskInfos.find((taskType) => taskType.type === type);
 
-  if (isLoading) {
-    return <LoadingScreen text={t("common:loading")} />;
+  switch (response.status) {
```

Comment:
Hey, do you agree that `response.status` makes the reader think this is related to an response status code e.g 200 or 500?

Reply:
Good point, I'll try to find another name for it.